docs/writing-a-benchmark.rst

```diff
@@ -240,10 +240,10 @@ passed to the ``Event`` function to ensure thread safety.
 
 Calling ``ctx.Conn.Execute`` executes the query against the database. If an
 error occurs, it is returned to mybench. For this workload, since there is only
-a single query.
+a single query.  // TODO: complete this sentence
```

Comment:
Need help here.

Reply:
I think we can just remove the word "since".

Reply:
Did a force-push with this edit.